### ISSUE TITLE
the charts are inverted

### ISSUE THREAD

**Issue by r-lyeh-archived** [mavam/compbench#3]
It seem the charts are inverted for all the images. Let me recapitulate:

1st img) Compression ratio is inverted. NONE should have compression ratio of 0 (null bar), whereas BROTLI11 (which has the highest compression ratio there, should be the largest bar)

2nd img) Chart is about speed, but looks like it is measuring time. So the higher time in the charts means the slowest, not the fastest. The compression speed vs decompression speed chart is an awesome idea for a chart indeed. Brotli11 and BSC (and specially ZPAQ which is missing in the charts btw) should be close to the (0,0) axis as they are among the slowest compressors.

3rd img) Again, the largest bars in here should be for LZ4 and/or LZ4HC which are among the fastest de/compressors. On the other hand, Brotli11/BSC/ZPAQ should be the smallest bars.

4th img) Similar issue

**Comment by mavam** [comment]
Great points, thanks for bringing them up!
1. By inverted, I assume you mean that the ratio `r` should be `1 - r` (as opposed to `1/x`), right? I'll use whatever the compression community uses, which I don't know too well.
2. It seems that speed is a misleading term due to multiple possible interpretations. For me, speed is velocity in the sense of distance (bytes) per unit time. How about calling it latency vs. throughput?
3. See (1).
4. Dito.

If you have other ideas for plots, please don't hesitate to get in touch.

Two more notes:
- In the latency vs. throughput tradeoff plot, I am taking the _sum_ of compression and decompression latency in order to get a single value. It could be worthwhile to have separate plots for compression and decompression in addition.
- I've added the missing ZPAQ to the list of examined algorithms.

**Comment by r-lyeh-archived** [comment]
aha good points. I assume that the larger the better always unless stated otherwise. These speed and compression ratio are better big, so they should match with the bars as well. I will think about it later. BTW, latest bundle has a couple of new algorithms. I've taken the liberty to link to your graphs in the frontpage. What's the licrnse of plot?

**Comment by mavam** [comment]
I've updated the plots according to the [official definitions](https://en.wikipedia.org/wiki/Data_compression_ratio) of _compression ratio_ and _space savings_. I've also normalized the speed values to MBs per seconds, which is probably a more reasonable definition of throughput. Finally, I added a `y = x` diagonal to the throughput scatterplot. All values below the diagonal have faster compression than decompression, which is only the case for ZPAQ and BSC. 

I've licensed the plot figures under Creative Commons, hope that works for you.
